gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Fri Aug 03 13:37:52 BST 2018
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
```

Comment:
why is this 4.10.2? this should either be updated to 6.x here or moved to a separate PR.

The rest of the 6.x changes are in this PR as far as I can tell but you've force pushed and messed with the commit history for some reason so I can't tell

Reply:
https://tools.hmcts.net/jira/browse/RIA-2951
is the story to upgrade gradle version.
For now I would like to push with these changes and I have informed Ravi

Reply:
You should revert the gradle wrapper changes then

Reply:
gradle version reverted to 4.10.2